src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -67,6 +67,17 @@ internal static bool TryConvertNodeToDPath(TexlBinding binding, DottedNameNode n
                 path = DPath.Root.Append(leftNodeName).Append(node.Right.Name);
                 return true;
             }
+            else if (node.Left is CallNode call)
+            {
+                var rightNodeName = node.Right.Name;
+                if (binding.TryGetReplacedIdentName(node.Right, out var rename))
+                {
+                    rightNodeName = new DName(rename);
+                }
+
```

Comment:
You can use function name here to add to DPath as we discussed offline.

Reply:
done